gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-bin.zip
```

Comment:
Note that using the [bugfix release 4.10.2](https://github.com/gradle/gradle/releases/tag/v4.10.2) over 4.10.1 or 4.10 is being recommended by the gradle team.